protocols/relay/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 - Update dependencies.
 
+- Implement `Debug` for `RelayHandlerEvent` and `RealyHandlerIn`. See [PR 2183].
```

Comment:
Minor typo `RealyHandlerIn`.


```suggestion
- Implement `Debug` for `RelayHandlerEvent` and `RelayHandlerIn`. See [PR 2183].
```

Reply:
Thanks. Fixed in https://github.com/libp2p/rust-libp2p/pull/2183/commits/6435bd743ecf63b7af93c706c3293c36ee71c7ad.